src/test/javascript/spec/service/metis/forwarded-message.service.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { ForwardedMessage } from 'app/entities/metis/forwarded-message.model';
+import { ForwardedMessageService } from '../../../../../main/webapp/app/shared/metis/forwarded-message.service';
+import { PostingType } from '../../../../../main/webapp/app/entities/metis/posting.model';
+
+describe('ForwardedMessageService', () => {
+    let service: ForwardedMessageService;
+    let httpMock: HttpTestingController;
+
+    const apiUrl = 'api/forwarded-messages';
+
+    const sampleForwardedMessage = new ForwardedMessage(1, 2, PostingType.POST, { id: 3 } as any, undefined, '');
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [HttpClientTestingModule],
+            providers: [ForwardedMessageService],
+        });
+
+        service = TestBed.inject(ForwardedMessageService);
+        httpMock = TestBed.inject(HttpTestingController);
+    });
+
+    afterEach(() => {
+        httpMock.verify();
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
+
+    describe('#createForwardedMessage', () => {
```

Comment:
I'm not sure if we have a strict naming convention here, but usually we don't use hashtags at the beginning.